shim/src/main.rs

```diff
@@ -20,13 +20,23 @@ use std::{
 };
 
 #[derive(Parser, Debug, Serialize, Default, PartialEq)]
-#[clap(author, version, about = "Turbocharge your monorepo", long_about = None, ignore_errors = true, disable_help_subcommand = true)]
+#[clap(author, about = "Turbocharge your monorepo", long_about = None)]
+#[clap(
+    disable_help_subcommand = true,
+    disable_help_flag = true,
+    disable_version_flag = true,
+    ignore_errors = true
+)]
 struct Args {
+    #[clap(long, short, global = true)]
+    help: bool,
+    #[clap(long, short, global = true)]
```

Comment:
The current Go code treats `-v` as verbosity and not version: [cmdutil.go](https://github.com/vercel/turborepo/blob/main/cli/internal/cmdutil/cmdutil.go#L131)
```suggestion
    #[clap(long, global = true)]
```